challenge-Programs/performance-challenge.md

```diff
@@ -16,58 +16,60 @@ We can’t wait to see innovative ideas sparkled! Participants are welcomed to j
+- The initial Contribution Credit of each individual participant or team is “0”**. The participants need to gain 400 Contribution Credits with “Easy” issues to be qualified to work on **Medium** or **Hard** issues.  
+- When the Contribution Credits owned by the participant is over "600" (including "600"),  "Easy" issues will be not valid for redeeming credits. 
+- TiKV is a big codebase that may require some time to get familiar with. Easy issues are a great way to start if you plan larger contributions in the future. If you have significant upstream experience (e.g. you wrote a gRPC library), talk to us and we can bypass this requirement.
 
-- Each individual participant or team can only pick up one Issue at a time. Folks working on larger issue might stumble into fixes for smaller, Easy, issues. That’s OK, submit those! But don’t overburden yourself, please!
+- Each individual participant or team can only pick up one issue at a time. Folks working on larger issues might stumble into fixes for smaller, Easy issues. That’s OK, submit these! And, don’t overburden yourself!
 
-### Step 3: Fix the Issue
-If you have any questions during the coding process, you can discuss with us through **#performance-challenge-program** channel, and the mentor assigned for the Issues will reply to you within 24h.
+### Step 3: Fix the issue
+If you have any questions during the coding process, you can discuss with us through **#performance-challenge-program** channel, and the mentor assigned for the issues will reply to you within 24h.
 
 ### Step 4: Commit your code
-When your solution is ready, you can open a PR and commit the code to the corresponding GitHub Repo (i.e.,tidb) as a GitHub Pull Request (aka PR). Remember to @ the mentor responsible of this Issue to review your PR in the comment after the PR is submitted. The Mentor will check your PR as soon as possible.
+When your solution is ready, commit the code and submit a PR and to the corresponding GitHub Repo (i.e., tidb). Remember to request the review from the mentor responsible for this issue. The mentor will check your PR as soon as possible.
 
-How to submit: When you finish the coding, you can submit the GitHub Pull Request (PR) to the corresponding Repo. About how to submit PR, see [GitHub Workflow](https://github.com/pingcap/community/blob/master/contributors/workflow.md). There is also [SRE-BOT Command](https://github.com/pingcap/community/blob/master/contributors/command-help.md) Help for your reference.
+How to submit:  see [GitHub Workflow](https://github.com/pingcap/community/blob/master/contributors/workflow.md). There is also [SRE-BOT Command](https://github.com/pingcap/community/blob/master/contributors/command-help.md) Help for your reference.
```

Comment:
* Seems like there is an extra space between the words "submit" and "see"
* The text of the link (https://github.com/pingcap/community/blob/master/contributors/command-help.md) is "SRE-BOT Command Help"

@BellaXiang @dcalvin PTAL